src/__tests__/index.js

```diff
@@ -272,6 +271,76 @@ const tests = {
       'somethingcontainedintheword',
     ],
   },
+  'takes startsWith and case into account': {
+    input: [
+      [
+        'somethingcontainedintheword',
+        'camelCaseContainedInTheWord',
+        'PascalCaseContainedInTheWord',
+        'kebab-case-contained-in-the-word',
+        'snake_case_contained_in_the_word',
+      ],
+      's',
+    ],
+    output: [
+      'snake_case_contained_in_the_word',
+      'somethingcontainedintheword',
+      'camelCaseContainedInTheWord',
+      'PascalCaseContainedInTheWord',
+      'kebab-case-contained-in-the-word',
+    ],
+  },
+  'takes case into account and ignores fake case': {
+    input: [
+      [
+        'startingWith_s',
+        'somethingcontainedintheword',
+        'camelCaseContainedInTheWord',
+        'PascalCaseContainedInTheWord',
+        'kebab-case-contained-in-the-word',
+        'snake_case_contained_in_the_word',
+        'fakeCase_one',
+        'fake_case-two',
+        'fake_caseThree',
+      ],
+      's',
+    ],
+    output: [
+      'snake_case_contained_in_the_word',
+      'startingWith_s',
+      'somethingcontainedintheword',
+      'camelCaseContainedInTheWord',
+      'PascalCaseContainedInTheWord',
+      'kebab-case-contained-in-the-word',
+      'fakeCase_one',
+      'fake_case-two',
+      'fake_caseThree',
+    ],
+  },
+  'takes case and acronym into account': {
+    input: [
+      [
+        'superduperfile',
+        'super-duper-file',
+        'super_duper_file',
+        'superDuperFile',
+        'SuperDuperFile',
+      ],
+      'sdf',
+    ],
+    output: [
+      'superDuperFile',
+      'SuperDuperFile',
+      'super-duper-file',
+      'super_duper_file',
+      // acronym
+      'superduperfile',
+    ],
+  },
+  'takes case acronym and equals into account': {
+    input: [['super-duper-file', 'sdff'], 'sdf'],
+    output: ['sdff', 'super-duper-file'],
```

Comment:
Actually, I want `super-duper-file` to win in this case.

Reply:
Alright, just to confirm, so that would mean that a `case acronym > starts with`:
 * CASE_SENSITIVE_EQUAL
 * EQUAL
 * **CASE ACRONYM**
 * STARTS_WITH
 * WORD_STARTS_WITH
 * STRING_CASE
 * CONTAINS
 * ACRONYM
 * MATCHES
 * NO_MATCH

Reply:
Ah, good point. Could we change this test to illustrate the point a little bit better? I think I want the ranking to be like this:

 * CASE_SENSITIVE_EQUAL
 * EQUAL
 * STARTS_WITH
 * WORD_STARTS_WITH
 * STRING_CASE
 * **CASE ACRONYM**
 * CONTAINS
 * ACRONYM
 * MATCHES
 * NO_MATCH